ome/omero-web
# group stats filter

Fixes bug reported at https://forum.image.sc/t/few-users-mistakenly-appear-in-the-statistics-of-all-groups/59932

To test:
 - Log in as a group owner. Under User Settings, look at the data usage for a group.
 - Shouldn't see any users listed who are not members of the group *unless* they have more than 10MB
 - Try removing someone from the group who has more than 10MB data, will see:

![Screenshot 2021-11-12 at 12 52 10](https://user-images.githubusercontent.com/900055/141470596-973612e6-9425-4aa0-b725-01da30fbfd74.png)

The query loads all Pixels and Files for *every* Experimenter in the DB, filtering by the current group context. This includes the User's Avatar photo which is in the "user" group (not filtered out by group context).
A much faster query is to only query the Experimenters who are members of the Group, but then we'd miss any user who has a lot of data in the group (but is not a member) e.g. previous member.
NB: the filtering of 10MB limit is done in JavaScript. All the query results are available in the JSON data.

Also update the flot library as I was getting poor layout with long user names.
